src/CoreWCF.Primitives/src/CoreWCF/Description/DispatcherBuilder.cs

```diff
@@ -590,6 +593,16 @@ private static void BuildDispatchOperation(OperationDescription operation, Dispa
 
                 parent.UnhandledDispatchOperation = child;
             }
+
+            var authorizationPolicyProvider = services.GetService<IAuthorizationPolicyProvider>();
+            if (authorizationPolicyProvider != null)
+            {
+                // TODO: Make this chain call async
+                Task<AuthorizationPolicy> getPolicyTask = operation.AuthorizeData.Count > 0
+                    ? AuthorizationPolicy.CombineAsync(authorizationPolicyProvider, operation.AuthorizeData)
```

Comment:
The `CombineAsync` call need a proper unit test that actually combine several policies set through multiple `AuthorizeAttribute`

Reply:
This is now done